### PR TITLE
Added set threshold value command 'Z'

### PR DESCRIPTION
**Pull request by maqoo**
communication e.g. "R0Z0C8\n"

### CODE REVIEW

File: Arduino/chorus-rf-laptimer/chorus-rf-laptimer.ino [hunk 1]
```diff
@@ -241,6 +242,8 @@ void setup() {
         pinMode(serialTimerPin, OUTPUT);
         pinMode(loopTimerPin, OUTPUT);
     );
+
+    Serial.write("X*\n");
```
Comment (voroshkov) [inline Arduino/chorus-rf-laptimer/chorus-rf-laptimer.ino]: what is the purpose of this?

File: Arduino/chorus-rf-laptimer/sendSerialHex.h [hunk 2]
```diff
@@ -35,6 +35,10 @@ uint8_t HEX_TO_BYTE (uint8_t hi, uint8_t lo) {
     return TO_BYTE(hi)*16+TO_BYTE(lo);
 }
 
+int16_t HEX_TO_UINT16 (uint8_t v4, uint8_t v3, uint8_t v2, uint8_t v1) {
```
Comment (voroshkov) [inline Arduino/chorus-rf-laptimer/sendSerialHex.h]: let's make it using 2 calls to HEX_TO_BYTE and shifting instead of multiplying with numeric constants

**Comment by anunique** [comment]
actually i've got another idea. what about checking the lenght of             case CONTROL_SET_THRESHOLD:

if its "R*S" its going to set to current value if its "R*S00ff" its going to set a given value using that 16 bit value

**Comment by maqoo** [comment]
I like the idea with 2 times hex_to_byte.

For the set threshold its also a nice idea to have it this way. I will prepare anaother commit. I've changed setting CONTROL_SET_SKIP_LAP0 with one byte and CONTROL_SET_SOUND also with one byte. So a command of 3 bytes in length would toggle it and if it has 4 bytes it would set the value. I've also change the MAX_MIN_LAP_TIME to 255 as it is the highest possible value to be set by app. I think 60 sec could be not enough.

**Comment by voroshkov** [comment]
Sorry, guys, cannot agree with you. I vote for distinct commands. It's less ambiguous and easier to parse.
Also I'd hold off changing anything in existing protocol unless really necessary.

**Comment by maqoo** [comment]
Then maybe a setter would be better rather than toggle in those places?

**Comment by anunique** [comment]
this one can be closed i guess.

setting of threshold is included in the last update!

**Comment by voroshkov** [comment]
Closed in release 0.6.1

**Pull request closed without merging**